deployer-image/Dockerfile

```diff
@@ -1,3 +1,15 @@
+FROM alpine AS test_schema_creator
+
+WORKDIR /app
+
+ARG GH_TOKEN
```

Comment:
revert this to older coder

Secret should be mounted in different way

https://github.com/appcd-dev/deployer/compare/f17b2c3cef1dbe4c006a75e7343ead77357414b0...0168c757b36b3ac3c30325db16522f38bb335c81#diff-4ec369aef76b3de9beb69cedae59c3dd4472738588d2b868bfddab3bb405d45cR5

Reply:
```
RUN --mount=type=secret,id=gh_token \
    apk add --no-cache yq 
```